pkg_resources depracation causing downstream CI failures

It looks like the `__init__.py` file here is using `pkg_resources`, which has been deprecated, and is now causing some downstream CI failures:

```
../../.tox/py311-test-alldeps-cov/lib/python3.11/site-packages/mpl_scatter_density/__init__.py:4: in <module>
    from pkg_resources import get_distribution, DistributionNotFound
../../.tox/py311-test-alldeps-cov/lib/python3.11/site-packages/pkg_resources/__init__.py:103: in <module>
    warnings.warn(
E   DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
```

I'm sure it's a quick fix but I don't have the bandwidth to submit a PR doing it right now.

--- a/mpl_scatter_density/__init__.py
+++ b/mpl_scatter_density/__init__.py
@@ -1,9 +1,7 @@
 from .scatter_density_artist import *  # noqa
 from .scatter_density_axes import *  # noqa
 
-from pkg_resources import get_distribution, DistributionNotFound
-
 try:
-    __version__ = get_distribution('mpl-scatter-density').version
-except DistributionNotFound:
+    from .version import version as __version__
+except ImportError:
     __version__ = 'undefined'

